Sources/Datadog/RUM/RUMVitals/VitalRefreshRateReader.swift

```diff
@@ -53,25 +55,53 @@ internal class VitalRefreshRateReader: ContinuousVitalReader {
         }
     }
 
+    // MARK: - Internal
+
+    internal func framesPerSecond(provider: FrameInfoProvider) -> Double? {
+        var fps: Double? = nil
+
+        if let lastFrameTimestamp = self.lastFrameTimestamp {
+            let currentFrameDuration = provider.currentFrameTimestamp - lastFrameTimestamp
+            guard currentFrameDuration > 0 else {
+                return nil
+            }
+            let currentFPS = 1.0 / currentFrameDuration
+
+            // ProMotion displays (e.g. iPad Pro and newer iPhone Pro) can have refresh rate higher than 60 FPS.
+            if let expectedCurrentFrameDuration = self.nextFrameDuration, provider.maximumDeviceFramesPerSecond > 60 {
```

Comment:
Nit - I think we can replace the 60 at the end of the if let as well 🙌

Reply:
good call, done.